client/src/components/Projects/ProjectEdit/index.jsx

```diff
@@ -98,13 +155,13 @@ export default class index extends Component {
               )}
             </label>
             <label className="main-label" htmlFor="name">
-              <h3 className="main-h3">Description :</h3>
-              {this.state.project.description && (
+              <h3 className="main-h3">dsescription :</h3>
+              {this.state.project.dsescription && (
                 <textarea
-                  name="description"
+                  name="dsescription"
                   onChange={this.handleOnChange}
                   className="main-input main-textArea"
-                  defaultValue={this.state.project.description}
+                  defaultValue={this.state.project.dsescription}
```

Comment:
typo! description not dsecription

Reply:
It is not my mistak, i tried to correct it but, i will make change also to backend... so, i left it as it is...

Reply:
@IsraaSulaiman  and @AymanAlqoqa 
we all should have reviewed it before, so it's our mistake all of us
`it's not my mistake` isn't good for teamwork

anyway, I agree with @AymanAlqoqa  we should do the refactor for the next week
today we need to finish up something working to deliver for our client tomorrow
plus don't resolve a conversation that you didn't start :wink:

Reply:
Ok